packages/dynamite/dynamite/lib/src/builder/client.dart

```diff
@@ -515,18 +516,9 @@ String buildParameterSerialization(
   }
 
   if (parameter.$in == openapi.ParameterType.header) {
-    final assignment = refer('_headers')
-        .index(literalString(parameter.pctEncodedName))
-        .assign(
-          refer('HeaderEncoder', 'package:dynamite_runtime/utils.dart')
-              .constInstance(const [], {
-                'explode': literalBool(parameter.explode),
-              })
-              .property('convert')
-              .call([refer(serializedName)]),
-        )
-        .statement
-        .accept(state.emitter);
+    final encoderRef = refer('HeaderEncoder', 'package:dynamite_runtime/utils.dart');
+    final assignment =
+        "_headers['${parameter.pctEncodedName}'] = ${allocate(encoderRef)}(explode: ${parameter.explode}).convert($serializedName);";
```

Comment:
I don't understand why this is better than before, especially because it doesn't use code_builder anymore?

Reply:
I have a local branch where ~60% of the client is generated with using code_builder but sadly this tanked the performance. 
I'd have to test it again but iirc the build time increased from ~1.5m to ~15m so I decided that this sadly isn't the way to go :(

I can keep it for now if you prefer.

Reply:
Ugh, then we should keep this change. Should be remove all other code_builder usage as well? Maybe that will make everything much faster?

Reply:
I'm not 100% sure. I think we'd need to conduct some further testing to be sure.
I'll keep the removal here for now as it also makes the code more readable.